backend/src/external_ip.rs

```diff
@@ -0,0 +1,16 @@
+//! Used to determine the internet address that connections from workers will appear to come from.
+//!
+//! For users writing scripts to access their infrastructure with firewalls requiring incoming
+//! connections to be from whitelisted IP addresses.
+
+use reqwest::Result;
+
+const WINDMILL: &'static str = "https://hub.windmill.dev/getip";
```

Comment:
This hostname is used in a few place, we should probably put it in a const file and have all relevant use-cases refer to it.